packages/docs/docs/welcome.md

```diff
@@ -2,24 +2,66 @@
 title: What is UniRep?
 ---
 
-# 👏 Welcome
+# Introduction
 
-<center><img src="/img/unirep-banner.png" alt="UniRep: Privacy &#x26; provable reputation" width="800px" /></center>
+UniRep is a zero-knowledge protocol that securely manages user data through anonymous identifiers, enabling trustless interactions and enhanced user privacy in applications. UniRep expands the notion of reputation to include various user data aspects, such as preferences, activity, alignments, and ownership.
 
-## Overview
+Using anonymous identifiers (epoch keys), the protocol allows for trustless engagement with applications while preserving user privacy. This approach promotes non-custodial applications that don't hold user data, reducing data breach risks and emphasizing security for both users and developers.
```

Comment:
I think the (epoch keys) can be linked to the protocol/epoch-key page
in line 21, the link can be removed

Reply:
added link to first paragraph, line 21